assets/sass/components/key-metrics/_googlesitekit-km-selection-panel.scss

```diff
@@ -101,6 +105,12 @@
 		flex-wrap: wrap;
 		justify-content: space-between;
 		margin-top: $grid-gap-desktop / 2;
+
+		@media (max-width: $bp-xsmallOnly) {
+			flex-direction: column;
+			margin-top: 0;
+			row-gap: 6px;
```

Comment:
I don't think the `row-gap` is needed with the spacing being applied using margin. WDYT?

Reply:
Thanks for spotting that. Totally missed it. You are correct